src/components/PromptForm/PromptForm.js

```diff
@@ -0,0 +1,43 @@
+import './PromptForm.css';
+import React, { Component } from "react";
+import React, { useState } from "react";
+
+class PromptForm extends Component {
+    constructor() {
+        super() 
+        this.state = {
+            prompt: '',
+            error: ''
+        }
+    }
+
+    handleChange = event => {
+        this.setState({ prompt: event.target.value });
+    }
+
+    clearInputs = () => {
+        this.setState({ prompt: '' });
+    }
+
+    render() {
+        return (
+            <div className="prompt-form-section">
+                <form>
+                <textarea className="writing-prompt" rows="10" cols="100">
```

Comment:
I haven't see "rows=10" or cols="100" before so I am interested in learning more about these